### PR TITLE
Polish the fs package

### PR DESCRIPTION
**Pull request by utkuufuk**

### CODE REVIEW

File: internal/fs/fs.go [hunk 1]
```diff
@@ -12,117 +12,103 @@ import (
...
 	fmt.Printf("setattr ino: %d\n", ino)
 
-	md, err := r.manager.GetMetadata(ino)
+	md, err := fs.manager.GetMetadata(ino)
 	if err != nil {
 		if err == common.ErrNotFound {
 			return nil, fuse.ENOENT
 		}
 
 		fmt.Fprintf(os.Stderr, "couldn't get metadata of inode %d: %v\n", ino, err)
```
Comment (utkuufuk) [inline internal/fs/fs.go:52]: @paddlesteamer Question; are these `fmt.Printf` and `fmt.Fprintf` statements are used/helpful in any way to the app or are they just for debugging?

File: internal/fs/fs.go [hunk 2]
```diff
@@ -12,117 +12,103 @@ import (
 
 type CloudStashFs struct {
 	manager *manager.Manager
-
```
Comment (paddlesteamer) [inline internal/fs/fs.go]: I think this empty line should stay, it separates the struct's own parameters and "inherited" parameters.

File: internal/fs/fs.go [hunk 3]
```diff
@@ -355,135 +327,119 @@ func (r *CloudStashFs) Read(ino int64, size int64, off int64, fi *fuse.FileInfo)
 	_, err = reader.Seek(off, 0)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "couldn't seek to provided offset %d: %v\n", off, err)
-
 		return nil, fuse.EIO
 	}
 
 	data := make([]byte, size)
-
```
Comment (paddlesteamer) [inline internal/fs/fs.go]: This should stay too. It looks crowded now.

**Review by paddlesteamer — COMMENTED** [review]
Other than that I like empty lines before `return`s too but I don't insist on them.

File: internal/fs/fs.go (hunk 2 above)
Comment (utkuufuk) [reply internal/fs/fs.go]: @paddlesteamer Makes sense, but I think a comment would be helpful indicating the inherited fields and how are they being used. I found it hard to understand what it stood for at the first glance and didn't realize that it was in fact an inherited field.

Comment (utkuufuk) [reply internal/fs/fs.go]: @paddlesteamer For instance, I don't see how `fuse.DefaultFileSystem` is being used, it doesn't seem to be referenced  from our code.

Comment (paddlesteamer) [reply internal/fs/fs.go]: Comments are nice of course, no argument against it but in `struct`s if it doesn't have a type it is inherited :D

Comment (paddlesteamer) [reply internal/fs/fs.go]: > @paddlesteamer For instance, I don't see how `fuse.DefaultFileSystem` is being used, it doesn't seem to be referenced from our code.

Ok, now I understand, I received a lot of questions about fuse implementation from you (about how nothing is referenced from our code), so let's take you [here](https://github.com/paddlesteamer/go-fuse-c/blob/master/example/hello/hello.go) and it's c [equivalent](https://www.maastaar.net/fuse/linux/filesystem/c/2016/05/21/writing-a-simple-filesystem-using-fuse/) to understand how fuse works. These are the tutorials I learned from.

**Review by paddlesteamer — APPROVED** [review]
:+1:

**Pull request merged**